docs/src/terminology.md

```diff
@@ -156,12 +160,24 @@ A [program](terminology.md#program) with the ability to interpret the binary enc
 
 ## lockout
 
-The duration of time for which a [validator](terminology.md#validator) is unable to [vote](terminology.md#ledger-vote) on another [fork](terminology.md#fork).
+The duration of time for which a [validator](terminology.md#validator) is unable to [vote](terminology.md#ledger-vote) on another [fork](terminology.md#fork) for risk of having their stake slashed.
+
+## lockup 
+
+Stake accounts can have a lockup which prevents the tokens they hold from being withdrawn before a particular date or epoch has been reached. While locked up, the stake account can still be delegated, un-delegated, or split, and its stake and withdraw authorities can be changed as normal. Only withdrawal into a wallet address is not allowed.
+
+## lockup authority 
+
+A lockup can be added when a stake account is first created, but it can be modified later, by the lockup authority or custodian, the address of which is also set when the account is created.
 
 ## native token
 
 The [token](terminology.md#token) used to track work done by [nodes](terminology.md#node) in a cluster.
 
+## neighborhoods
```

Comment:
This is only used in Turbine, so fine if that's only defined within that one document. Also, please avoid definitions in the plural form.